cmd/tikv-ctl/src/executor.rs

```diff
@@ -1008,15 +1008,20 @@ impl DebugExecutor for DebugClient {
             ),
             ("paused", resp.get_region_read_progress_paused().to_string()),
             ("discarding", resp.get_discard().to_string()),
-            // TODO: figure out the performance impact here before implementing it.
-            // (
-            //     "duration to last update_safe_ts",
-            //     format!("{} ms", resp.get_duration_to_last_update_safe_ts_ms()),
-            // ),
-            // (
-            //     "duration to last consume_leader_info",
-            //     format!("{} ms", resp.get_duration_to_last_consume_leader_ms()),
-            // ),
+            (
+                "duration to last update_safe_ts",
```

Comment:
How many places might update them without the `now` parameter? Should we consider call it "approximate duration to ..."?

Reply:
As far as I can see, they're only cdc and backup-stream. Let's call it "... from resolved-ts routine"